instance/tests/integration/integration_instance.py

```diff
@@ -42,6 +43,9 @@
 
 # Tests #######################################################################
 
+@override_settings(
+    DEFAULT_LMS_PREVIEW_DOMAIN_PREFIX='integ-',
```

Comment:
@itsjeyd Even if it doesn't have any now, I think it's good practice to ensure we only deprovision the VMs we know have been provisioned by the integration tests. Otherwise someone might unsuspectingly create a VM there, and end up losing work because of it.

And +1 for addressing the issue pointed out by @mtyaka , if possible as part of this task, to not have to go through the overhead of creating/assigning/reviewing a whole new task again for this.